### PR TITLE
chore: #2253 smb ui tidy up

### PR DESCRIPTION
**Pull request by phmngocnghia**
# Pull request checklist

## Does this close any currently open issues?

fixes: #2253

**Please check if your PR fulfills the following requirements:**

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint`) has passed locally and any fixes were made for failures
- [x] Test (`yarn test`) has passed locally and any fixes were made for failures

## Pull request type

**Please check the type of change your PR introduces:**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Update UI

## What is the current behavior?

Issue Number: #

## What is the new behavior?
image
image
image

-

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/elements/src/styles/components/spreadsheet.scss [hunk 1]
```diff
@@ -153,12 +153,12 @@ span.data-grid-container:focus {
 
     &.read-only {
       color: $black;
```
Comment (willmcvay) [inline packages/elements/src/styles/components/spreadsheet.scss:155]: Small thing but can we make the text grey for read-only? So it looks "disabled" - use the same grey as form disabled field text - thx

File: packages/smb-onboarder/src/components/ui/negotiators-list/negotiator-status-checkbox.tsx [hunk 2]
```diff
@@ -89,7 +90,7 @@ export const NegotiatorStatusCheckbox: React.FC<NegotiatorStatusCheckboxProps> =
   const checkBoxId = `${row}-${col}`
 
   return (
-    <div className="field field-checkbox">
+    <div className={`field field-checkbox ${spreadSheetCheckbox}`}>
```
Comment (willmcvay) [inline packages/smb-onboarder/src/components/ui/negotiators-list/negotiator-status-checkbox.tsx:93]: Can we use cx here?

File: packages/smb-onboarder/src/constants/routes.ts [hunk 3]
```diff
@@ -1,14 +1,13 @@
 const Routes = {
   LOGIN: '/login',
-  HOME: '/home',
+  HELP: '/help',
   OFFICES: '/offices',
   USERS: '/users',
   SOURCES: '/sources',
```
Comment (willmcvay) [inline packages/smb-onboarder/src/constants/routes.ts:6]: Routes from sources down can be removed now right?

File: packages/smb-onboarder/src/core/app.tsx [hunk 4]
```diff
@@ -10,8 +10,8 @@ import UploadProvider from '@/components/providers/upload-provider'
 
 export const globals = css`
   :global() {
-    body {
-      background-color: unset;
+    html {
+      font-size: 14px;
```
Comment (willmcvay) [inline packages/smb-onboarder/src/core/app.tsx:14]: This is inherited from Elements I think. Anyway, should be 16px unless desktop when it's 14px. See my comment in index.tsx, we should get this for free now

File: packages/smb-onboarder/src/core/index.tsx [hunk 5]
```diff
@@ -24,6 +24,13 @@ window.reapit = {
 
 export const renderApp = (Component: React.ComponentType) => {
   const rootElement = document.querySelector('#root') as Element
+
+  const isDesktop = getMarketplaceGlobalsByKey()
+  const html = document.querySelector('html')
+  if (isDesktop && html) {
+    html.classList.add('is-desktop')
```
Comment (willmcvay) [inline packages/smb-onboarder/src/core/index.tsx:31]: This is correct - will handle the toggle of 14px, 16px rem base

**Comment by willmcvay** [comment]
Just an additional comment, on the offices page, there is loads of whitespace below the pagination. Pagination should be in it's own section and look like the developer apps page eg 

Screenshot 2020-08-05 at 09 49 15

**Comment by phmngocnghia** [comment]
@willmcvay, I have updated based on your suggestions. Please, help me to re-review the PR again. Thanks

**Pull request merged**